ci/playbooks/kuttl/e2e-kuttl.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Bootstrap step
+  ansible.builtin.import_playbook: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/ci_framework/playbooks/01-bootstrap.yml"
+
+- name: Deploy and run KUTTL operator tests
```

Comment:
Note: I'm not 100% sure the ansible.cfg will know about the location of the generated install_yamls_makes role. Maybe something to consider and pass down from the job definition? Please check the provided ansible.cfg in the repository for the proper location.